docs/source/user/extensions.rst

```diff
@@ -433,6 +357,122 @@ all jupyterlab organization extensions.
      ]
    }
 
+Managing Plugins with Plugin Manager
+------------------------------------
+
+Each JupyterLab extension is composed of one or more plugins. Plugins are also
+used by JupyterLab core itself.
+
+The Advanced Plugin Manager enables listing and disabling individual plugins,
+but may have been disabled by your system administrator.
+
+Funcitonally, toggling an extension from the plugin manager is equivallent to
+running ``jupyter labextension enable/disable``.
+
+
+.. image:: ../images/plugin-manager-search-notebook.png
+   :align: center
+   :class: jp-screenshot
+   :alt: An example search result in the plugin extension listing.
+
+Locking and Unlocking Plugins
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Administrators may locked specific plugins with:
+
+.. code:: bash
+
+    jupyter labextension lock my-extension:plugin
+
+To unlock a locked plugin:
+
+.. code:: bash
+
+    jupyter labextension unlock my-extension:plugin
+
+The locked plugins appear on the pluigin list with a lock icon and cannot be enabled/disabled from the user interface:
+
+.. image:: ../images/plugin-manager-plugin-locked.png
+   :align: center
+   :class: jp-screenshot
+   :alt: An example row of the plugin list with `@jupyterlab/pluginmanager-extension:plugin` plugin locked.
+
+
+Managing Extensions with ``jupyter labextension``
```

Comment:
This was moved from above mostly without modification, please see comment below highlighting the place which got changed. The reasons for moving it around:
- we want users to use the Extension Manager rather than `jupyter labextension` if possible, so let's have order: Extension Manager → Plugin Manager → Command Line Interfaces
- there is a section about `jupyter labextension install` below, it was strangely separated from the rest of `jupyter labextension` section